Traceback (most recent call last):
  File "/root/Mewtwo/game/toontown/uberdog/ServerStart.py", line 30, in <module>
    run()
  File "game/otp/ai/AIBase.py", line 175, in run
    self.taskMgr.run()
  File "/usr/share/panda3d/direct/task/Task.py", line 538, in run
    self.step()
  File "/usr/share/panda3d/direct/task/Task.py", line 492, in step
    self.mgr.poll()
  File "/usr/share/panda3d/direct/distributed/ConnectionRepository.py", line 609, in readerPollUntilEmpty
    while self.readerPollOnce():
  File "/usr/share/panda3d/direct/distributed/ConnectionRepository.py", line 616, in readerPollOnce
    self.handleDatagram(self.private__di)
  File "game/toontown/distributed/ToontownInternalRepository.py", line 43, in handleDatagram
    AstronInternalRepository.handleDatagram(self, dgi)
  File "/usr/share/panda3d/direct/distributed/AstronInternalRepository.py", line 244, in handleDatagram
    self.dbInterface.handleDatagram(msgType, di)
  File "/usr/share/panda3d/direct/distributed/AstronDatabaseInterface.py", line 299, in handleDatagram
    self.handleCreateObjectResp(di)
  File "/usr/share/panda3d/direct/distributed/AstronDatabaseInterface.py", line 73, in handleCreateObjectResp
    self._callbacks[ctx](doId)
  File "game/toontown/uberdog/ExtAgent.py", line 825, in createLoginResponse
    self.loginAccount(account, clientChannel, accountId, playToken)
TypeError: loginAccount() takes exactly 7 arguments (5 given)

